frontends/web/src/routes/settings/settings.tsx

```diff
@@ -93,6 +97,17 @@ class Settings extends Component<Props, State> {
     });
   };
 
+  private handleBlkExpFormChange = (event: React.SyntheticEvent) => {
+    const target = (event.target as HTMLInputElement);
+    if (target.name !== 'blockExplorerURL') {
+      return;
+    }
+    this.setState({
+      [target.name]: target.value,
+      restart: false,
```

Comment:
remove this? if the restart flag is set it means a restart is needed. Changing the block explorer form does not change that.